vmdb/db/migrate/20150630025128_create_tenant.rb

```diff
@@ -0,0 +1,17 @@
+class CreateTenant < ActiveRecord::Migration
+  def change
+    create_table :tenants do |t|
+      t.string :domain
+      t.string :subdomain
+      t.string :customer_name
+      t.string :vmdb_name
```

Comment:
Again, can you describe how vmdb_name is different from customer_name?  I just don't know what the two column represent, which is why I think they're hard to name.